src/main/java/Bigmouth.java

```diff
@@ -0,0 +1,204 @@
+import java.io.*;
+import java.net.URL;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+public class Bigmouth {
+    private static final DateTimeFormatter INPUT_FORMAT = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+    private static final URL fileURL = Bigmouth.class.getProtectionDomain().getCodeSource().getLocation();
+    private static String path = fileURL.getPath(); //FILE_PATH;
+    public static String rootPath = path.substring(0, path.indexOf("ip") + 3) +  "/data/Bigmouth.txt";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm");
+    private static final DateTimeFormatter DISPLAY_FORMATTER = DateTimeFormatter.ofPattern("MMM dd yyyy, h:mma");
+
+    public static void main(String[] args) {
+        loadFromFile();
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Bigmouth\nWhat can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        while (true) {
+            String userInput = scanner.nextLine();
+
+            try {
+                if (userInput.equals("bye")) {
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" Bye. Hope to see you again soon!");
+                    System.out.println("____________________________________________________________");
+                    break;
+                }
+
+                if (userInput.equals("list")) {
+                    if (tasks.isEmpty()) {
+                        throw new BigmouthException("Your task list is empty!");
+                    }
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" Here are the tasks in your list:");
+                    for (int i = 0; i < tasks.size(); i++) {
+                        System.out.println(" " + (i + 1) + "." + tasks.get(i));
+                    }
+                    System.out.println("____________________________________________________________");
+                } else if (userInput.startsWith("mark ")) {
+                    int taskNumber = Integer.parseInt(userInput.split(" ")[1]) - 1;
+                    if (taskNumber < 0 || taskNumber >= tasks.size()) {
+                        throw new BigmouthException("Invalid task number. Please enter a valid task number.");
```

Comment:
This looks like it might exceed the character limit per line, consider refactoring it!